README.md

```diff
@@ -11,7 +11,7 @@
     <a href="https://pypi.org/project/scivision/">
         <img alt="PyPI" src="https://img.shields.io/pypi/v/scivision">
     </a>
-    <a href="https://mybinder.org/v2/gh/alan-turing-institute/scivision/main?labpath=examples%2Fpretrained_imagenet_scivision_example.ipynb">
+    <a href="https://mybinder.org/v2/gh/alan-turing-institute/scivision/0.1.2?labpath=pretrained_imagenet_scivision_example.ipynb">
```

Comment:
please fix the `labpath` as you removed the examples folder

```suggestion
    <a href="https://mybinder.org/v2/gh/alan-turing-institute/scivision/0.1.2?labpath==examples%2Fpretrained_imagenet_scivision_example">
```

Reply:
In 0.1.2 the notebook is in the root of the repository (we hadn't yet moved it to examples)

Reply:
@ots22 it makes sense now. I'll cancel the suggestion and accept the changes.